### PR TITLE
Changed all Gtk.VBox and Gtk.HBox instances to Gtk.Box with a Gtk.Orientation

### PR DESCRIPTION
**Pull request by tristan957**
VBox and HBox became deprecated at least in GTK 3.22. I replaced them with their parent class Gtk.Box. Gtk.Box takes as an argument a Gtk.Orientation which can either be VERTICAL or HORIZONTAL.

If any changes need to be made, please let me know.

### CODE REVIEW

**Comment by ikeydoherty** [comment]
Erm - I'm seeing .pot changes.. ? and a version change? :)

**Comment by tristan957** [comment]
Looking into the .pot changes although I definitely didn't change it brb.

**Comment by tristan957** [comment]
Alright I reverted the version change which was just a dumb mistake. .pot should be the same as master branch. Try this on for size

**Comment by tristan957** [comment]
Something seems to be up with lines around 683 in updates_view.py

**Comment by tristan957** [comment]
Ok sorry for all the extra commits at the end but everything should be up to date with master except for lines where HBox and VBox are mentioned. Hopefully you can merge this smoothly

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 setup(
     name            = "solus-sc",
-    version         = "15.0",
+    version         = "14.0",
```
Comment (alarsyo) [inline setup.py]: Problem is that now your pull request changes the version again, this time Ikey did bump it to 15.0 in e8ac37dcf7511990125b7f964e474d5e593dc5d1

**Comment by tristan957** [comment]
I'm quite shite at git. pls forgive

**Comment by alarsyo** [comment]
I think you should `git rebase` your pull request, currently it's going to create a mess in the commit history :P

**Comment by tristan957** [comment]
Can you help me use the command? Never done it before

**Comment by tristan957** [comment]
Tempted to just start over because I know this is ugly but I don't know how to resolve it

**Comment by ikeydoherty** [comment]
Ok start over once my changes are in :)

**Comment by tristan957** [comment]
Want me to start a new pull request? This was garbage and very early in my git experience. Sorry about that!

**Pull request closed without merging**